cmd/components_paths.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/chanzuckerberg/fogg/config"
+	"github.com/chanzuckerberg/fogg/errs"
+	"github.com/chanzuckerberg/fogg/plan"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	componentsCmd.AddCommand(componentsListCmd)
+	componentsListCmd.Flags().StringP("config", "c", "fogg.yml", "Use this to override the fogg config file.")
+}
+
+var componentsListCmd = &cobra.Command{
+	Use:           "paths",
+	Short:         "List paths for all configured components.",
+	SilenceErrors: true, // If we don't silence here, cobra will print them. But we want to do that in cmd/root.go
+	RunE: func(cmd *cobra.Command, args []string) error {
+		var e error
+		// Set up fs
+		pwd, e := os.Getwd()
```

Comment:
```suggestion
		pwd, err := os.Getwd()
```
I think `err` is more "idiomatic" + consistent with our go code